docs/reference/content/driver-reactive/getting-started/quick-start-primer.md

```diff
@@ -0,0 +1,118 @@
++++
+date = "2015-03-17T15:36:56Z"
+title = "Quick Start Primer"
+[menu.main]
+  parent = "MongoDB Reactive Streams"
+  identifier = "Primer"
+  weight = 20
+  pre = "<i class='fa'></i>"
++++
+
+# Quick Start Primer
+
+The aim of this guide is to provide background about the MongoDB Reactive Streams driver and its asynchronous API before going onto 
+looking at how to use the driver and MongoDB.
+
+{{% note %}}
+See the [installation guide]({{< relref "driver-reactive/getting-started/installation.md" >}})
+for instructions on how to install the MongoDB Reactive Streams Driver.
+{{% /note %}}
+
+## Reactive Streams
+
+This library is an implementation of the [reactive streams](http://www.reactive-streams.org) specification and the reactive stream API 
+consists of the following components:
+
+1. Publisher
+2. Subscriber
+3. Subscription
+
+A `Publisher` is a provider of a potentially unbounded number of sequenced elements, published according to the demand received from it's `Subscriber(s)`.
+
+In response to a call to `Publisher.subscribe(Subscriber)` the possible invocation sequences for methods on the `Subscriber` are given by the following protocol:
+
+```
+onSubscribe onNext* (onError | onComplete)?
+```
+
+This means that `onSubscribe` is always signalled, followed by a possibly unbounded number of `onNext` signals (as requested by `Subscriber`) 
+followed by an `onError` signal if there is a failure, or an `onComplete` signal when no more elements are available—all as long as 
+the `Subscription` is not cancelled.
+
+For more information about reactive streams go to: [http://www.reactive-streams.org](http://www.reactive-streams.org).
+
+
+## From Async Callbacks to Subscribers
```

Comment:
Not sure if this title makes sense.  We don't mention callbacks anywhere in the text.

Reply:
Done